node/node.go

```diff
@@ -156,6 +157,9 @@ type Node struct {
 	// SectorBuilder is used by the miner to fill and seal sectors.
 	sectorBuilder sectorbuilder.SectorBuilder
 
+	// PeerTracker maintains a list of peers good for fetching.
+	PeerTracker *net.PeerTracker
```

Comment:
Can we make this an interface instead? I think it should make changes easier down the road.

Reply:
I want to follow the dominant pattern which is still concrete structs.  Are you talking about refactoring node construction for better mocking?  I can see that point of view but don't know about concrete plans to make that refactor.  If we do it should be easy enough to convert this one to an iface along with the 20 other concrete structs if we do that.

Reply:
I don't view that as the dominant pattern here, a majority of these fields are interfaces (over half). But as you have states we don't have a concrete plan for a node refactor. I'll leave this up to you.

Reply:
I think concrete types are ok here, for now, but the various components that get references to these objects should primarily get interfaces. This could change after we clean things up a lot here.